src/WireMock.Net/Serialization/MappingConverter.cs

```diff
@@ -173,6 +187,49 @@ public string ToCSharpCode(IMapping mapping, MappingConverterSettings? settings
         return sb.ToString();
     }
 
+    private string ConvertJsonToAnonymousObjectDefinition(JToken token, int ind = 0)
+    {
+        string FormatObject(JObject jObject)
+        {
+            var indStr = new string(' ', 4 * ind);
+            var indStrSub = new string(' ', 4 * (ind+1));
```

Comment:
can you do a format document in visual studio?

Reply:
Fixed